tests/e2escenarios/core_beta_flow.go

```diff
@@ -33,9 +33,9 @@ var _ = Describe("Core beta flow", func() {
 		oc = helper.NewOcRunner("oc")
 		odo = "odo"
 
-		project = helper.CreateRandProject()
 		context = helper.CreateNewContext()
 		os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
+		project = helper.CreateRandProject()
```

Comment:
Out of curiosity, what impact does this have whether we create a project earlier or later?

Reply:
It is just to make sure that the set global preference path is used in the project creation step though there is no validation done for it. But if you keep the project create step as it was then project creation step picks the default preference path (~/.odo/preference.yaml) and then the very next odo command start picking the preference path from newly set path. 

So i cleaned up this in all test file and later i will create some fresh spec where the validation will be done including all possible cases without duplicating the scenario.

Reply:
This is for another discussion, but I think that `CreateRandProject` should be using `oc` not `odo`.